pkg/lib/clients/transformers.go

```diff
@@ -0,0 +1,12 @@
+package clients
+
+import (
+	"k8s.io/client-go/rest"
+)
+
+func SetWarningHandler(wh rest.WarningHandler) ConfigTransformer {
+	return ConfigTransformerFunc(func(config *rest.Config) *rest.Config {
+		config.WarningHandler = wh
```

Comment:
nit: should we check for a nil config here?

Reply:
I updated the doc comment to indicate that the argument will never be nil and that implementations should never return nil.